app/views/notifications/index.html.erb

```diff
@@ -1,22 +1,7 @@
 <div class="notifications-container">
 
-  <div class="notifications-listing">
-    <h4>Notifications</h4>
-    <% @unapproved_sign_slides.each do |ss| %>
-      <div id="unapproved-sign-slide-<%= ss.id %>" class="unapproved-sign-slide">
-        <p><%= image_tag ss.slide.screenshot, class: 'screenshot' %></p>
-        <p>The slide <%=link_to ss.slide.menu_name, ss.slide %> needs to be approved to go on the sign <%= link_to ss.sign.name, ss.sign %>.</p>
-        <p>
-          <%= link_to 'Approve', approve_sign_slide_path(ss), remote: true, method: :post, class: 'approve-sign-slide btn green' %>
-          <%= link_to 'Reject', reject_sign_slide_path(ss), remote: true, method: :delete, class: 'reject-sign-slide btn red' %>
-        </p>
-      </div>
-    <% end %>
-  </div>
+  <%= render 'notifications' %>
```

Comment:
I see that you are trying to use minimal html here, but I try to only split out a partial like this if it is actually being re-used somewhere (or if its in a loop).  

If you want to render the notifications in a partial, then I would move the loop out of the partial, then call the partial from the loop.

Maybe like....
```erb
<h4>Notifications</h4>
<%= render partial: 'unapproved_sign_slides', collection: @unapproved_sign_slides, as: :ss %>
```

then have the partial be:
```erb
    <div id="unapproved-sign-slide-<%= ss.id %>" class="unapproved-sign-slide">		
 -        <p><%= image_tag ss.slide.screenshot, class: 'screenshot' %></p>		
 -        <p>The slide <%=link_to ss.slide.menu_name, ss.slide %> needs to be approved to go on the sign <%= link_to ss.sign.name, ss.sign %>.</p>		
 -        <p>		
 -          <%= link_to 'Approve', approve_sign_slide_path(ss), remote: true, method: :post, class: 'approve-sign-slide btn green' %>		
 -          <%= link_to 'Reject', reject_sign_slide_path(ss), remote: true, method: :delete, class: 'reject-sign-slide btn red' %>		
 -        </p>		
 -      </div>
```